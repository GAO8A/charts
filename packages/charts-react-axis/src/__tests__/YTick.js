import React from 'react';
import renderer from 'react-test-renderer';
import YTick from '../YTick';

describe('YTick Component', () => {
  it('should render', () => {
    const tree = renderer.create(
      <YTick offset={250} label="Label" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
