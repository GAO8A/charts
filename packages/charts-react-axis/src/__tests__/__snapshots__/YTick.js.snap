exports[`YTick Component should render 1`] = `
<g
  className="tick"
  transform="translate(0,250)">
  <line
    stroke="#000"
    x2="-6"
    y1="0.5"
    y2="0.5" />
  <text
    dy="0.32em"
    fill="#000"
    x="-9"
    y="0.5">
    Label
  </text>
</g>
`;
